contrib/epee/include/net/levin_protocol_handler_async.h

```diff
@@ -907,6 +917,10 @@ bool async_protocol_handler_config<t_connection_context>::for_connection(const b
   async_protocol_handler<t_connection_context>* aph = find_connection(connection_id);
   if (!aph)
     return false;
+  if (!aph->start_outer_call())
```

Comment:
This should also be unnecessary. The `m_connects_lock` provides protects other threads from destroying the object, and the callback cannot destroy the connection object safely.

Reply:
Disregard.